### ISSUE TITLE
[Test Maps] `chaos`

### ISSUE THREAD

**Issue by snake-biscuits** [snake-biscuits/bsp_tool#72]
- [x] Tools:
   * Momentum Mod Playtest `0.9.4` [there is a waiting list]
   - [x] Level Editor(s):
      * Hammer
      * [Hammer++ (CS:GO)](https://ficool2.github.io/HammerPlusPlus-Website/download.html)
        [Setup Guide](https://docs.momentum-mod.org/guide/hammer-plus-plus-setup/)
   - [x] Compiler(s): Source SDK, [BSPConversionLib](https://github.com/momentum-mod/BSPConversionLib)
   * Documentation / Guides:
     * [VDC: CS:GO Level Creation](https://developer.valvesoftware.com/wiki/Counter-Strike:_Global_Offensive_Level_Creation)
     * [Momentum Mod Docs](https://docs.momentum-mod.org/)
 

 - [x] Map(s):
   - [x] Basic room with spawnpoint & lighting
      `mapsrc/Hammer/Momentum Mod/mp_lobby.map` -> `maps/Momentum Mod/mp_lobby.bsp`
     - [x] `sources.txt` crediting mapper / contributor
     - [x] Compiled
     - [x] `.bsp.info.txt` file with compile log (`.log` in `mapsrc`)
     - [x] Tested

> NOTE: converted `q3/chronic` w/ `BSPConversionLib --newbsp` & it threw no loading errors, unsure if that was a fluke
> quake3 conversion seems to break texture projection, might look into fixes

Active Mods / Modding Community (Optional):
  * [VDC](https://developer.valvesoftware.com/wiki)
  * [MomentumMod](https://momentum-mod.org/)

**Comment by snake-biscuits** [comment]
Attempting to run maps from console results in this error:
![momentum_mod_buffer_overflow](https://user-images.githubusercontent.com/36507175/206083516-cff307d5-e529-4563-8a3c-dbad1a8ae382.png)
Might be related to absence of `.zon` zone file?

bhop training runs without one though, but I can't leave the first room; no clue as to why.

**Comment by snake-biscuits** [comment]
This crash appears to be an issue with the current version of momentum mod;
In the meantime, a Hammer++ guide has been released:
https://docs.momentum-mod.org/guide/hammer-plus-plus-setup/

**Comment by snake-biscuits** [comment]
The 4088 bytes overflow error has been addressed in Momentum Mod Playtest `0.9.4`
`tests/maps/Momentum Mod/mp_lobby` now launches (using `-mapping` in launch options)
No map changes were required, `mp_lobby` just runs now

**Comment by snake-biscuits** [comment]
Source Engine based branch_scripts require more complex maps
We need to test at least 1 of each lump we have mapped

TL;DR add a map with displacements
